src/mame/drivers/kaypro.cpp

```diff
@@ -8,7 +8,11 @@
     and 1x sio. One of the sio ports communicated with the keyboard with a coiled
     telephone cord, complete with modular plug on each end. The keyboard carries
     its own Intel 87C51 processor and is an intelligent device.
-
+	
+	The Kaycomp Kay Pro V / Non-Linear Systems Kaypro 5: https://retrocmp.de/kaypro/kay-p1_vers.htm#kay-5, a version of the Kaypro II, 
+	but with a 5MB harddisk instead of one of the floppy drives, was announced in December 1982, but apart from the relevant articles of the time,
+	only one photo can be found.
+	
```

Comment:
At the very least, you should wrap this addition at the same column as the rest of the comment.